assets/widget/uexSina.html

```diff
@@ -57,6 +57,7 @@
 	function sharePic(){
         var content = "中国最大的移动中间平台AppCan对新浪微博分享的图片支持测试";
         var realImgPath = "res://sotower.png";//http:// 本地
+        var realImgPath = "wgts://sotower.png";

```

Comment:
you should Commented out the first `var realImgPath` line code in javascirpt. @BadWaka